thrift/src/test/java/com/linecorp/armeria/common/thrift/text/StackWalkingCurrentThriftMessageClassFinderTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.thrift.text;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.function.Supplier;
+
+import javax.annotation.Nullable;
+
+import org.apache.thrift.TException;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.Test;
+
+import com.linecorp.armeria.common.thrift.text.RpcDebugService.doDebug_args;
+import com.linecorp.armeria.common.util.SystemInfo;
+
+class StackWalkingCurrentThriftMessageClassFinderTest {
```

Comment:
I see code duplication between this test and DefaultCurrentThriftMessageClassFinderTest. How about merging them into one by using parameterized tests? (You could provide 2 finder instances for Java 9+ and only the default one for Java 8.)